internal/handlers/common/aggregations/expression.go

```diff
@@ -69,23 +70,20 @@ func (e *ExpressionError) Code() ExpressionErrorCode {
 
 // Expression is an expression constructed from field value.
```

Comment:
> Expression is an expression

We can't define the term using the term itself.

Compare and contrast with `types.Path` description